peekingduck/pipeline/nodes/dabble/trackingv1/tracking_files/opencv_tracker.py

```diff
@@ -0,0 +1,151 @@
+# Copyright 2021 AI Singapore
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tracking-by-detection using OpenCV's MOSSE Tracker."""
+
+from typing import Any, Dict, List, NamedTuple
+
+import cv2
+import numpy as np
+
+from peekingduck.pipeline.nodes.dabble.trackingv1.tracking_files.utils import (
+    iou_candidates,
+    xyxyn2tlwh,
+)
+
+
+class OpenCVTracker:  # pylint: disable=too-few-public-methods
+    """OpenCV's MOSSE tracker.
+
+    Attributes:
+        is_initialised (bool): Flag to determine this is the first run of the
+            tracker.
+        iou_threshold (float): Minimum IoU value to be used with the matching
+            logic.
+        next_track_id (int): ID for the next (unmatched) object detection to be
+            tracked.
+        tracks (Dict[int, Track]): Maps track IDs to their respective
+            tracker and bbox coordinates.
+    """
+
+    def __init__(self) -> None:
+        self.is_initialised = False
+        self.iou_threshold = 0.1
```

Comment:
Can this be a tweakable param in config file?

Reply:
Added